index.bs

```diff
@@ -46,6 +46,22 @@ url: https://dom.spec.whatwg.org/#ref-for-concept-getelementsbytagname; type: in
 
 <h2 id="basic-principles">Principles behind design of Web APIs</h2>
 
+<h3 id="priority-of-constituencies">Priority of constituencies</h3>
+
+In case of conflict,
+consider users over authors over implementors over specifiers over theoretical purity.
+In other words costs or difficulties to the user
+should be given more weight than costs to authors;
+which in turn should be given more weight than costs to implementors;
+which should be given more weight than costs to authors of the spec itself,
```

Comment:
```suggestion
which should be given more weight than costs to authors of the specification itself,
```
It seems it was also spec in the original, but I think it's better to not use abbreviations.